packages/testsuite/cypress/e2e/elytron/test-configuration-subsystem-elytron-global-settings.cy.ts

```diff
@@ -35,6 +35,7 @@ describe("TESTS: Configuration => Subsystem => Security => Settings => Global Se
 
   it("Edit default-authentication-context", () => {
     cy.navigateTo(managementEndpoint, "elytron");
+    cy.help();
```

Comment:
I'm OK with having this as a workaround but it should be documented here as a workaround.

Reply:
I have added this comment before help call

`// this call is just workaround to prevent of occuring JBEAP-25005`

I believe this thread can be resolved.